mintapi/api.py

```diff
@@ -147,7 +147,12 @@ def get_accounts(self, get_detail=False):  # {{{
     def get_transactions(self):
         if not pd:
             raise ImportError('transactions data requires pandas')
-        from StringIO import StringIO
+
+        try:
+            from StringIO import StringIO  # Python 2
+        except ImportError:
+            from io import StringIO  # Python 3
```

Comment:
Why not change this to:
```
except ImportError:
    from io import BytesIO as StringIO
```

Then you could change 167 - 169 to simply be:
```
s = StringIO(result.content)
```
and it would work all the time.